api/monitor.py

```diff
@@ -194,7 +209,7 @@ def __init__(self, _db, collection, lookup=None, provider=None):
     def endpoint(self):
         return self.lookup.metadata_needed()
 
-    def run_once(self, start, cutoff):
+    def run_once(self, progress):
```

Comment:
Here, we don't actually use the thing passed in to `run_once`, so it's just a matter of changing the signature.